services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerMultiClusterConfig.java

```diff
@@ -286,4 +286,8 @@ public long getServiceDiscoveryRegistrationRetryMS() {
   public List<String> getControllerInstanceTagList() {
     return getCommonConfig().getControllerInstanceTagList();
   }
+
+  public boolean getUpdateRealTimeTopic(String clusterName) {
```

Comment:
Do we need this method?  If yes let's update it's name to be consistent with the new name: isHybridStorePartitionCountUpdateEnabled